lib/avalon/batch/manifest.rb

```diff
@@ -25,14 +25,51 @@ class Manifest
       SKIP_FIELDS = [:collection]
 
       def_delegators :@entries, :each
-      attr_reader :spreadsheet, :file, :name, :email, :entries, :package
+      attr_reader :spreadsheet, :file, :name, :email, :entries, :package, :manifest_logger
 
       class << self
         def locate(root)
+          locate_with_status(root)[:processable]
+        end
+
+        def locate_with_status(root)
+          status = {
```

Comment:
Can you be in all these "statuses" at the same time? By making this a hash of arrays you get this impression that you can be in multiple states of processing/processed/error/processable all at the exact same time?

No need for this hash/array stuff unless you can be in multiple states at the same time and it matters that you are. Even if you technically can...does this matter for any reason? From the other code that depends on this, it seems like we always logically process this by checking the statuses one by one, starting with...are you an error?

Why not just check...am I error? Yes...okay return status of error. No?..check if im processing? Yes...then I am in processing status...etc...etc..etc...else therefore im in a processable status. And status that is returned in just a symbol/string

This will simplify the rest of your code that is depending on the statuses as a result.